assets/src/edit-story/elements/video/frame.js

```diff
@@ -14,14 +14,68 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import { useState } from 'react';
+import styled from 'styled-components';
+
 /**
  * Internal dependencies
  */
 import StoryPropTypes from '../../types';
 import MediaFrame from '../media/frame';
+import { elementFillContent } from '../shared';
+import { useStory } from '../../app/story';
+import { ReactComponent as PlayIcon } from '../../icons/play.svg';
+import { ReactComponent as PauseIcon } from '../../icons/pause.svg';
+
+const Play = styled(PlayIcon)`
+  width: 48px;
+  height: 48px;
+`;
+const Pause = styled(PauseIcon)`
+  width: 48px;
+  height: 48px;
+`;
+
+const Wrapper = styled.div`
+  ${elementFillContent}
+`;
+
+const ButtonWrapper = styled.div`
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  cursor: pointer;
+`;
 
 function VideoFrame({ element }) {
-  return <MediaFrame element={element} />;
+  const { id, isPlaying } = element;
+  const [hovered, setHovered] = useState(false);
+
+  const {
+    actions: { updateElementById },
+  } = useStory();
+
+  const handlePlayPause = () => {
+    updateElementById({ elementId: id, properties: { isPlaying: !isPlaying } });
+  };
+
+  return (
+    <Wrapper
+      onPointerEnter={() => setHovered(true)}
+      onPointerLeave={() => setHovered(false)}
```

Comment:
This works a little weird. The enter/leave happens at weird times, it seems. Doesn't happen at the edge of the video directly, only on some inner shape? Maybe it's because I have other elements with other frames overlapping it though - might want to talk to @wassgha or @dvoytenko  about how this works in the layer ordering.
![hovervideo](https://user-images.githubusercontent.com/637548/77378036-47a21200-6d4b-11ea-90bb-40a1365bbd4a.gif)

Reply:
Saw that, It's a drop indicator, I think we should deal with it after the merge (fixes from @wassgha) maybe it will be fixed already.